### ISSUE TITLE
VIsnet deployed models and scorers?

### ISSUE THREAD

**Issue by hmacdope** [choderalab/asapdiscovery#969]
@kaminow, @fyng

What is the current status of our Visnet models. Could we deploy one for general use in the repo at this stage? If that is the case I can also add a scorer and fold them into the project support workflows.  No stress if they are not ready, just wanted to check in see how this is going.

**Comment by kaminow** [comment]
I don't think we have one trained and ready to go, but I think code-wise we have the capabilities. probably though I'd say this isn't necessarily a high priority, since I think we understand this model even less than the other structure-based ones at the moment, so not sure that we're in a good spot science-wise to be deploying this for the med chemists. but happy to bump it up in priority if you think it's important @hmacdope
